src_code/src/controllers/services/Inventory/EquipmentAddition.js

```diff
@@ -411,6 +479,6 @@ function PurchaseDateValidator(res, purchaseDate){
 
 /** Exports the module/functions */
 module.exports = {
-    AddTool,
-    AddToolValidation
+    EquipmentAddition: EquipmentAddition,
+    EquipmentAdditionValidation: EquipmentAdditionValidation

```

Comment:
You can just let it be EquipmentAddition if you want :) since they are both the same.